packages/editor/src/plugins/text/components/suggestions.tsx

```diff
@@ -1,65 +1,126 @@
-import IconFallback from '@editor/editor-ui/assets/plugin-icons/icon-fallback.svg'
+import { EditorInput } from '@editor/editor-ui/editor-input'
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
 import { useEditorStrings } from '@serlo/frontend/src/contexts/logged-in-data-context'
-import { cn } from '@serlo/frontend/src/helper/cn'
+import React, { useEffect } from 'react'
+import { Key } from 'ts-key-enum'
 
-import type { SuggestionOption } from '../hooks/use-suggestions'
+import { SuggestionItem } from './suggestion-item'
+import { SuggestionOption } from '../hooks/use-suggestions'
 
 interface SuggestionsProps {
-  options: SuggestionOption[]
-  suggestionsRef: React.MutableRefObject<HTMLDivElement | null>
-  selected: number
-  onMouseDown: (pluginType: string) => void
-  onMouseMove: (index: number) => void
+  basicOptions: SuggestionOption[]
+  interactiveOptions: SuggestionOption[]
+  searchInputRef: React.MutableRefObject<HTMLInputElement | null>
+  searchString: string
+  setSearchString: (searchString: string) => void
+  itemRefs: React.MutableRefObject<(HTMLButtonElement | null)[]>
+  currentlyFocusedItem: number
+  setCurrentlyFocusedItem: (index: number) => void
+  insertSelectedPlugin: (pluginType: EditorPluginType | string) => void
 }
 
-export const Suggestions = ({
-  options,
-  suggestionsRef,
-  selected,
-  onMouseDown,
-  onMouseMove,
-}: SuggestionsProps) => {
+export function Suggestions(props: SuggestionsProps) {
+  const {
+    basicOptions,
+    interactiveOptions,
+    searchInputRef,
+    searchString,
+    setSearchString,
+    itemRefs,
+    currentlyFocusedItem,
+    setCurrentlyFocusedItem,
+    insertSelectedPlugin,
+  } = props
+
   const editorStrings = useEditorStrings()
 
-  if (options.length === 0) {
-    return <div>{editorStrings.plugins.text.noItemsFound}</div>
+  useEffect(() => {
+    const focusTimeout = setTimeout(() => {
+      searchInputRef.current?.focus()
+    }, 0)
+    return () => clearTimeout(focusTimeout)
+  }, [searchInputRef])
```

Comment:
If this is needed it means the `autoFocus` on the `EditorInput` does not work, right?
Do you figure out why?

Reply:
Thats correct, it doesn't work here. 

I assume it's related to editor focus hacks, or because it's inside a modal. I've created a [ticket](https://linear.app/serlo/issue/PE-64/investigate-autofocus-in-editorinput) in Linear to investigate.